sandbox-backend/src/main/resources/db/migration/V0000001_addEmployee.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO employee VALUES (null, 'E00001', 'pass1', 'USER', 1, null, null);
```

Comment:
TODO: Passwords should be encrypted, and it would be better to use inserts in format 
``` INSERT INTO table (columns) VALUES (values)``` At this case you can omit null values